try:
    # 执行语句
    print('')
except TypeError as ty:
    pass
except:
    print('捕获异常')
